CMakeLists.txt

```diff
@@ -203,6 +203,7 @@ export(EXPORT portblas
 
 option(BLAS_ENABLE_TESTING "Whether to enable testing" ON)
 option(ENABLE_EXPRESSION_TESTS "Whether to build expression tree fusion tests" OFF)
+option(ENABLE_JOINTMATRIX_TESTS "Whether to build joint_matrix GEMM tests" OFF)
```

Comment:
Thanks for your PR, @muhammad-tanvir-1211.

Probably this flag needs to be documented in the [README.md#cmake-options](https://github.com/codeplaysoftware/portBLAS/blob/master/README.md#cmake-options).

Reply:
This has been added in the README.